shenyu-plugin/shenyu-plugin-response/src/main/java/org/apache/shenyu/plugin/response/strategy/WebClientMessageWriter.java

```diff
@@ -50,16 +49,10 @@
         return chain.execute(exchange).then(Mono.defer(() -> {
             ServerHttpResponse response = exchange.getResponse();
             ClientResponse clientResponse = exchange.getAttribute(Constants.CLIENT_RESPONSE_ATTR);
-            if (Objects.isNull(clientResponse)
```

Comment:
why remove this

Reply:
![image](https://user-images.githubusercontent.com/2174082/159616500-cac0c113-d3f1-4867-8921-090574d88d6b.png)
this is the current implementation for `WebClientPlugin` that overrides the business response status code. and `NettyClientMessageWriter` does not cover the business response status and status code. so I change this code with the same logic.

Reply:
if the business responds to the 500 or 502 or 504,  the business data will be covered by Shenyu.